tillerinobot/src/main/java/tillerino/tillerinobot/lang/Norwegian.java

```diff
@@ -0,0 +1,271 @@
+			.then(new Action("klem " + apiUser.getUserName()));
+	}
+
+	@Override
+	public String help() {
+		return "Hei! Jeg er en robot som drepte Tillerino og tok over kontoen hannes. Bare tuller med deg, men jeg bruker kontoen ofte selv."
+				+ " [https://twitter.com/Tillerinobot Status og oppdateringer]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki Kommandoer]"
+				+ " - [http://ppaddict.tillerino.org/ ppaddict]"
+				+ " - [https://github.com/Tillerino/Tillerinobot/wiki/Contact Kontakt]";
+	}
+
+	@Override
+	public String faq() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ Ofte stilte spørsmål]";
+	}
+
+	@Override
+	public String featureRankRestricted(String feature, int minRank, OsuApiUser user) {
+		return "Beklager, for øyeblikket er " + feature + " bare tilgjengelig for spillere som har nådd rank: " + minRank + ".";
+	}
+
+	@Override
+	public String mixedNomodAndMods() {
+		return "Hva mener du med nomod med mods?";
+	}
+
+	@Override
+	public String outOfRecommendations() {
+		return "[https://github.com/Tillerino/Tillerinobot/wiki/FAQ#the-bot-says-its-out-of-recommendations-what-do"
+				+ " Jeg har anbefalt alt jeg kan tenke over.]."
+				+ " Prøv andre anbefalinger eller !reset. Hvis du er usikker, skjekk !help.";
+	}
+
+	@Override
+	public String notRanked() {
+		return "Ser ut som mappen ikke er ranked.";
+	}
+
+	@Override
+	public boolean isChanged() {
+		return false;
+	}
+
+	@Override
+	public void setChanged(boolean changed) {
+
+	}
+
+	@Override
+	public String invalidAccuracy(String acc) {
+		return "Unkjent accuracy: \"" + acc + "\"";
+	}
+
+	@Override
+	public Response optionalCommentOnLanguage(OsuApiUser apiUser) {
+		/*
+		 * TRANSLATION NOTE: This line is sent to the user right after they have
+		 * chosen this Language implementation. The English version refers to
+		 * itself as the default version ("just the way I am"), so translating
+		 * the English message doesn't make any sense.
+		 * 
+		 * Instead, we've been using the line
+		 * "*Translator* helped me learn *Language*." in translations. Replace
+		 * *Translator* with your osu name and *Language* with the name of the
+		 * language that you are translating to, and translate the line into the
+		 * new language. This serves two purposes: It shows that the language
+		 * was changed and gives credit to the translator.
+		 * 
+		 * You don't need to use the line above, and you don't have have to give
+		 * yourself credit, but you should show that the language has changed.
+		 * For example, in the German translation, I just used the line
+		 * "Nichts leichter als das!", which translates literally to
+		 * "Nothing easier than that!", which refers to German being my first
+		 * language.
+		 * 
+		 * Tillerino
+		 * 
+		 * P.S. you can put a link to your profile into the line like this:
+		 * [https://osu.ppy.sh/u/2070907 Tillerino]
+		 */
+		return new Message("Så du liker meg slik som jeg er? :)");

```

Comment:
this message shouldnt be translated directly, read the comments above it. You don't have to do the suggested thing of crediting yourself but translating it directly doesnt make sense imo